templates/quickstart-jira-dc.template.yaml

```diff
@@ -535,7 +535,7 @@ Parameters:
     Default: "8.5.6"
     AllowedPattern: '(\d+\.\d+\.\d+(-?.*))|(latest)'
     ConstraintDescription: Must be a valid version number or 'latest'; for example, 8.1.0 for Jira Software, or 4.1.0 for Jira Service Management.
-    Description: The version of Jira Software or Jira Service Desk to install. If choosing Jira All, pick the version of Jira Software. TODO CHECK IF THIS IS CORRECT Find valid versions at https://confluence.atlassian.com/x/TVlNLg (Jira Software), https://confluence.atlassian.com/x/jh9-Lg (Jira Service Desk), or https://confluence.atlassian.com/x/XM2EO (Atlassian Enterprise Releases).
+    Description: The version of Jira Software or Jira Service Desk to install. If choosing Jira All, pick the version of Jira Software and we will automatically select the latest compatible version of Jira Service Management. Find valid versions at https://confluence.atlassian.com/x/TVlNLg (Jira Software), https://confluence.atlassian.com/x/jh9-Lg (Jira Service Desk), or https://confluence.atlassian.com/x/XM2EO (Atlassian Enterprise Releases).
```

Comment:
I suggest to rephrase to: `and the latest compatible version of Jira Service Management will be selected automatically`

Reply:
I will make this change after reply to my comment so I can do both at once :)